### ISSUE TITLE
[Question] How to enable maven version range analysis?

### ISSUE THREAD

**Issue by jamesmmchugh** [renovatebot/config-help#234]
### Which Renovate are you using? CLI, App, or Pro
Docker OSS

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps
GitHub Enterprise

### What is your question?

The following dependency in my maven pom:

```
		<dependency>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok</artifactId>
			<version>[1.12.6, 1.18.6]</version>
		</dependency>
```

Isn't picked up for updating by Renovate even though the latest version is 1.18.8, which falls outside the current range. I noticed that Renovate should support this (https://github.com/renovatebot/renovate/pull/3206). Am I missing some configuration in order for this to be picked up?

**Comment by jamesmmchugh** [comment]
Never mind, had a space in the version range which I could see from debug logs meant it was not picked up as a valid version. Removed space character and works now. However the onboarding PR is quoting two updates:

The expected one:
Update dependency org.projectlombok:lombok to v1
Schedule: "at any time"
Branch name: renovate/org.projectlombok-lombok-1.x
Upgrade org.projectlombok:lombok to [1.12.6,1.18.8]

And one I was not expecting:
Update dependency org.projectlombok:lombok to [1.12.6,0.12.0]
Schedule: "at any time"
Branch name: renovate/org.projectlombok-lombok-0.x
Upgrade org.projectlombok:lombok to [1.12.6,0.12.0]

Is anyone able to explain this behaviour?

**Comment by rarkins** [comment]
It’s a big. I’m looking at it now

**Comment by rarkins** [comment]
Should be fixed in renovate@17.3.4

**Comment by jamesmmchugh** [comment]
@rarkins awesome, cheers for the quick responses
